src/Bridge/Doctrine/Orm/Filter/NumericFilter.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ApiPlatform\Core\Bridge\Doctrine\Orm\Filter;
+
+use ApiPlatform\Core\Bridge\Doctrine\Orm\Util\QueryNameGenerator;
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Doctrine\ORM\QueryBuilder;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+/**
+ * Filter by a numeric value.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ * @author Amrouche Hamza <hamza.simperfit@gmail.com>
+ */
+class NumericFilter extends AbstractFilter
+{
+    /*
+     * Type of numeric in Doctrine see http://doctrine-orm.readthedocs.org/projects/doctrine-dbal/en/latest/reference/types.html
+     *
+     */
+    const DOCTRINE_NUMERIC_TYPES = [
+        'bigint' => true,
```

Comment:
I'd prefer using the class constants:

```php
// use Doctrine\DBAL\Types\Type as DBALType;

const DOCTRINE_NUMERIC_TYPES = [
    DBALType::BIGINT => true,
    DBALType::DECIMAL => true,
    DBALType::FLOAT => true,
    DBALType::INTEGER => true,
    DBALType::SMALLINT => true,
];
```

sorted, and `time` is not a numeric type, according to the linked Doctrine reference.